Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Timing_HeartBeat.names

```diff
@@ -1,7 +1,7 @@
 {
     "entries": [
         {
-            "base": "{BA107060-249D-4818-9CEC-7573718273FC}",
+            "base": "{E73DB180-A325-763B-A1FE-517B548AF66E}",
```

Comment:
So the missing header is caused by the wrong uuid? assume the rest change is not related to GHI?

Reply:
I regenerated the file, I hadn't regenerated it after making changes to the generation algorithm, so it picked up a few improvements as well